daisy_workflows/build-publish/rhui/health_checks/rhui-health-check.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Executes RHUI health check
+
+[Service]
+Type=oneshot
+ExecStart=python3 /opt/google-rhui-infra/health_check.py \
+    --node rhua \
```

Comment:
the name of this file is generic but it specifies rhua here, do we want to dupe this file for cds? or parameterize it in some way?

Reply:
I thought you'd spot that :)  you're right -- each node type should be correctly represented here. do you have a preference on the approach? (dupe versus a template versus something else)